travel_platform/travel_app/views.py

```diff
@@ -3,10 +3,28 @@
 
 from django.http import Http404
 
+from django.http import JsonResponse
+
 from django.shortcuts import render
 
 from .models import Query
 
 
 def index(request):
     return render(request, 'travel_app/main.html')
+
+
+def coord(request):
+    """
+    View added for checking of interaction of map from landing page with 
+    server.
+    :param request: 
```

Comment:
If you don't expect to describe all arguments and return, please remove these lines.